netbench/netbench-collector/src/main.rs

```diff
@@ -18,24 +36,363 @@ pub struct Args {
 
     #[structopt(long, short, parse(try_from_str=parse_duration), default_value = "1s")]
     pub interval: Duration,
+
+    #[structopt(long, short)]
+    pub coordinate: bool,
+
+    #[structopt(long, required_if("coordinate", "true"))]
+    pub server_location: Option<String>,
+
+    #[structopt(long, required_if("coordinate", "true"))]
+    pub client_location: Option<String>,
+
+    #[structopt(long, required_if("coordinate", "true"))]
+    pub run_as: Option<String>,
```

Comment:
I think it might be nicer to have this as a server or client enum, and `location()` and `other_location()` could be methods implemented on that enum.